### DISCUSSION TITLE
Infracost directly populate JSON into DynamoDB

### DISCUSSION THREAD

**Discussion by pazarr** [infracost/infracost#1893] [Ideas & Feature Requests]
I think it would be awesome if _infracost_ can push JSON outputs into DynamoDB without having a middle ware. This can make output within the pipeline more accessible. It could be used for reporting and BI tools to find trends etc.
I.E.
1) Running CI/CD on CodeBuild for terraform PRs. Infracost generate the cost implication for the new piece of IaC and using AWS role assigned to codebuild, it updates DynamoDB table.
2) Running CI/CD on Jenkins/Bamboo/CircleCI for terraform PRs. Infracost generate the cost implication for the piece of IaC and using secret credentials to update DynamoDB table with the JSON.

**Comment by alikhajeh1** [comment]
Hi @pazarr! We recently released the initial version of [Infracost Cloud](https://www.infracost.io/pricing/), where the Infracost CLI can be configured to send its JSON output to your organization in that product from all of your CI/CD pipelines.

You can follow its [Get started guide](https://www.infracost.io/docs/infracost_cloud/get_started/) to try it. We're interested in working with users to see what kinds of reports/charts they want to generate from that data, but also look into providing data exports to other BI tools they want to use directly. Ping me at ali.hosseini@infracost.io if you want a demo, happy to show you some wireframes too of things we're working on right now.
